File: UI/src/main/java/ca/warp7/rt/feature/dataset/DatasetFeature.java
package ca.warp7.rt.feature.dataset;

import ca.warp7.rt.core.feature.Feature;
import ca.warp7.rt.core.feature.FeatureItemTab;

import java.util.Collections;
import java.util.List;

public class DatasetFeature implements Feature {
    @Override
    public List<FeatureItemTab> getLoadedTabs() {
        return Collections.singletonList(new FeatureItemTab(
                "Dataset Options", "fas-database:16:gray", getFeatureId(),
                FeatureItemTab.Group.SingleTab, ""
        ));
    }

    @Override
    public String getFeatureId() {
        return "dataset";
    }
}

File: UI/src/main/kotlin/ca/warp7/rt/ext/dataset/DatasetFeature.kt
package ca.warp7.rt.ext.dataset

import ca.warp7.rt.core.feature.Feature
import ca.warp7.rt.core.feature.FeatureItemTab
import ca.warp7.rt.core.feature.FeatureLink

class DatasetFeature : Feature {

    override val link get() = FeatureLink("Dataset Options", "fas-database", 16)

    override val loadedTabs
        get() = listOf(FeatureItemTab(
                "Dataset Options", "fas-database:16:gray", featureId,
                FeatureItemTab.Group.SingleTab, ""
        ))

    override val featureId get() = "dataset"
}
